ElectronClient/gui/utils/NoteListUtils.js

```diff
@@ -126,9 +126,22 @@ class NoteListUtils {
 						}
 						clipboard.writeText(links.join(' '));
 					},
-				})
+				}),
 			);
 
+			if (noteIds.length <= 1) {
+				menu.append(
+					new MenuItem({
+						label: _('Pin Item'),
```

Comment:
"Pin note" to make it consistent with existing context menu items.

Reply:
Changed it to "Pin / Unpin note" to allow toggle.

Reply:
Is it possible to switch it on context?